tools/src/main/java/org/apache/kafka/tools/ShareGroupsCommand.java

```diff
@@ -68,6 +71,9 @@ public static void run(ShareGroupCommandOptions opts) {
             // Currently the tool only supports listing of share groups
             if (opts.options.has(opts.listOpt))
```

Comment:
Consistent formatting for the entire if-else-if please.

Reply:
will fix, guess it was part of previous PR